scan/src/pages/BlockHomePage.re

```diff
@@ -87,32 +110,45 @@ let make = () => {
               | None => ""
               }
             }
+            size=Text.Md
+            weight=Text.Thin
+            spacing={Text.Em(0.06)}
+            color=Colors.grayHeader
+            nowrap=true
           />
         </div>
       </Col>
     </Row>
     <VSpacing size=Spacing.xl />
     <THead>
       <Row>
-        <Col> <div className=Styles.icon /> </Col>
+        <Col> <HSpacing size=Spacing.xl /> </Col>
```

Comment:
I think padding on the left and right side of the head table feel different from design